gateway/js/app.js

```diff
@@ -980,7 +1023,17 @@ function main(){
 			loadJS("js/forms.js");
 			loadJS("js/base.js");
 			loadHTML("html/footer.html");
-			formSubmission('#signup','create_user','User created - Please check your email','User exist');
+			//formSubmission('#signup','create_user','User created - Please check your email','User exist');
+			document.getElementById("signup-btn1").onclick = function () {
+				//location.href = "https://auth-itg.hpe.com/signin/register";
+				if (signupwindowObjectReference == null || signupwindowObjectReference.closed){
+					var URL = 'https://auth.hpe.com/signin/register'
```

Comment:
@susmithp @vejmarie not for this PR but this will be refactored in future to be configurable w/o hardcoded instance.